.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,92 @@
+name: Bug report
+description: Create a report to help us improve CodeIgniter
+title: "Bug: "
+labels: ['bug']
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Before you begin, please
+        ensure that there are no existing issues, whether still open or closed, related
+        to your report. If there is, your report will be closed promptly.
+
+  - type: input
+    id: php-version
+    attributes:
+      label: 'PHP Version'
+    validations:
+      required: true
+
+  - type: input
+    id: codeigniter-version
+    attributes:
+      label: 'CodeIgniter4 Version'
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: operating-systems
+    attributes:
+      label: Which operating systems have you used?
+      description: You may select more than one.
+      options:
+        - label: macOS
+        - label: Windows
+        - label: Linux
+    validations:
+      required: true
+
+  - type: dropdown
+    id: server
+    attributes:
+      label: Which SAPI did you use?
```

Comment:
Will people understand SAPI? Would "web server software" or something be more clear?